tests/tc_template_bang_completion/expected.txt

```diff
@@ -0,0 +1,2 @@
+calltips
+this(T data)
```

Comment:
this doesn't seem right, shouldn't the auto-completions rather be
```
Wrapper(T)
Something(T, X)
```
?

(since you want to know what template parameter you are filling out right now.

It's not possible to manually specify template arguments to constructors, so it also doesn't need any special case there.